api/api.py

```diff
@@ -197,18 +196,12 @@ def dispatcher(router, request, response):
             response.headers['Content-Type'] = 'application/json; charset=utf-8'
     except webapp2.HTTPException as e:
         util.send_json_http_exception(response, str(e), e.code)
-    except Exception as e: # pylint: disable=broad-except
-        if config.get_item('core', 'debug'):
-            message = traceback.format_exc()
-        else:
-            message = 'Internal Server Error'
-        util.send_json_http_exception(response, message, 500)
```

Comment:
webapp2 doesn't define a default handler for any error codes, so it will call [WSGIApplication._internal_error](https://github.com/GoogleCloudPlatform/webapp2/blob/master/webapp2.py#L1582) which logs to the root logger with level error, however it only logs the exception and not a traceback.  I'm going to removed #451 from the list of issues this PR fixes, and update it to include an action item for adding a 500 handler to log the traceback.